Controller/DrawingObject/FieldLineDrawing.py

```diff
@@ -17,107 +20,46 @@ def draw(self, painter):
-            painter.drawRect(x_goal, y_goal, goal_depth_resize, goal_width_resize)
-            x_goal = x + length
-            y_goal = y + width / 2 - goal_width_resize / 2
-            painter.setPen(QtToolBox.create_pen(color=(255, 255, 0),
-                                                style='SolidLine',
-                                                width=3))
-            painter.drawRect(x_goal, y_goal, goal_depth_resize, goal_width_resize)
-
             painter.setPen(QtToolBox.create_pen(color=(255, 255, 255),
                                                 style='SolidLine',
-                                                width= QtToolBox.field_ctrl.line_width * QtToolBox.field_ctrl.ratio_screen))
-
-            # Rectangle de contour du terrain
-            painter.drawRect(x, y, length, width)
-
-            # Ligne de la surface de réparation
-            rad_width = QtToolBox.field_ctrl.defense_radius * QtToolBox.field_ctrl.ratio_screen
-            x_rad_goal = x
-            y_rad_goal = y + width / 2 - QtToolBox.field_ctrl.defense_stretch / 2 * QtToolBox.field_ctrl.ratio_screen
-            # => TopLeft
-            painter.drawArc(QtCore.QRect(x_rad_goal - rad_width,
-                                         y_rad_goal - rad_width,
-                                         rad_width * 2,
-                                         rad_width * 2),
-                            0, 90 * 16)
+                                                width=QtToolBox.field_ctrl.line_width * QtToolBox.field_ctrl.ratio_screen))
 
-            y_rad_goal = y + width / 2 + QtToolBox.field_ctrl.defense_stretch / 2 * QtToolBox.field_ctrl.ratio_screen
-            # => BotLeft
-            painter.drawArc(QtCore.QRect(x_rad_goal - rad_width,
-                                         y_rad_goal - rad_width,
-                                         rad_width * 2,
-                                         rad_width * 2),
-                            270 * 16, 90 * 16)
-            # => TopRight
-            x_rad_goal = x + length
-            y_rad_goal = y + width / 2 - QtToolBox.field_ctrl.defense_stretch / 2 * QtToolBox.field_ctrl.ratio_screen
-            painter.drawArc(QtCore.QRect(x_rad_goal - rad_width,
-                                         y_rad_goal - rad_width,
-                                         rad_width * 2,
-                                         rad_width * 2),
-                            90 * 16, 90 * 16)
-            # => BotRight
-            x_rad_goal = x + length
-            y_rad_goal = y + width / 2 + QtToolBox.field_ctrl.defense_stretch / 2 * QtToolBox.field_ctrl.ratio_screen
-            painter.drawArc(QtCore.QRect(x_rad_goal - rad_width,
-                                         y_rad_goal - rad_width,
-                                         rad_width * 2,
-                                         rad_width * 2),
-                            180 * 16, 90 * 16)
+            # Dessine tous les arcs
```

Comment:
I don't think we need that comment, the code is pretty self-explanatory (for once)